local-setup/README.md

```diff
@@ -0,0 +1,36 @@
+#  How to use the local-setup
+
+## Prerequisite
+- worker built with `make`
+- integritee-node built with `cargo build --release`
+
+## Steps
+Adapt or create your own config file, as in the example of `simple-config.json`. Be mindful of the ports in case you're running the script on a server multiple people are working on.
+
+### Launch worker and node in terminal one
+You can launch the workers and the node with:
+```bash
+cd <worker directory>
+./local-setup/launch.py ./local-setup/simple-config.json
+```
+wait a little until all workers have been launched. You can stop the worker and node simply by pressing `Ctrl + c`.
+
+### Open a second terminal to show logs
+```bash
+cd <worker directory>/local-setup
+# If you work with docker: exec into the running container:
+docker exec -it [container-id] bash
+cd work
+# run the tmux logger script with:
+./tmux_logger.sh
```

Comment:
This should not work like that. IMO the correct sequence is:

```
# If you work with docker: exec into the running container:
docker exec -it [container-id] bash
cd work/local-setup
# run the tmux logger script with:
./tmux_logger.sh
```

Reply:
Which documentation are you basing this on? What is the `work` directory in the docker?

If this documentation mounts the integritee-worker root directory into the `work` directory of the container, we don't need to enter the docker at all. I remember being wrong about that, when I wrote the script's documentation.

I think you can run the `tmux_logger.sh` from the host system.

Reply:
I'm not using docker often. Wrote it after the documentation in https://github.com/capsule-corp-ternoa/worker/blob/develop/local-setup/README.md . Which might be outdated.
So ... you mean 
docker exec -it [container-id] bash
./local-setup/./tmux_logger.sh

is working in docker?

Reply:
Ok, I see. Then we don't need to enter the docker to run the logger. :) Hence, the steps to run the logger is simply:

```
cd local-setup
./tmux_logger.sh
```

And it does not matter if you work with docker or not.

Reply:
Done